packages/react/src/components/Cta/Cta.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ArrowDown20, ArrowRight20, Launch20 } from '@carbon/icons-react';
+import { ButtonGroup } from '../../patterns/sub-patterns/ButtonGroup';
+import { CardLink } from '../../patterns/sub-patterns/CardLink';
+import { FeaturedLink } from '../../patterns/blocks/FeaturedLink';
+import { LinkWithIcon } from '../LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * CTA component
+ *
+ * @param {object} props props object
+ * @param {string} props.style style ( text | card | button | feature ).
+ * @param {object} props.type tyle ( jump | local | external ).
+ * @param {object} props.cta cta object which includes href,text, copy, card and image, button  properties.
+ * @returns {*} CTA Component
+ */
+const CTA = ({ style, type, ...cta }) => {
+  if (style === 'card') {
+    if (type === 'jump') {
+      return (
+        <CardLink
+          {...cta}
+          icon={iconSelector(type)}
+          onClick={e => {
+            handleOnClick(e, cta.id);
+          }}
+        />
+      );
+    }
+    return <CardLink {...cta} icon={iconSelector(type)} />;
```

Comment:
Wondering if we could simplify a lot of this logic with a simple tertiary, and only retain a single component to return?

```jsx
<CardLink
  {...cta}
  icon={iconSelector(type)}
  onClick={type === 'jump' ? handleOnClick : null}
  }}
/>
```

Reply:
Okay